DataFormats/FTLDigi/interface/FTLDigiCollections.h

```diff
@@ -3,10 +3,15 @@
 
 #include "DataFormats/Common/interface/SortedCollection.h"
 #include "DataFormats/FTLDigi/interface/FTLDataFrameT.h"
-#include "DataFormats/ForwardDetId/interface/FastTimeDetId.h"
+#include "DataFormats/ForwardDetId/interface/BTLDetId.h"
+#include "DataFormats/ForwardDetId/interface/ETLDetId.h"
 #include "DataFormats/FTLDigi/interface/FTLSample.h"
 
-typedef FTLDataFrameT<FastTimeDetId,FTLSample> FTLDataFrame;
```

Comment:
Not sure if it's okay to delete these...

Reply:
we want to keep backward compatibility to read old data, so it is better to keep this (although practically useless)